woocommerce-services.php

```diff
@@ -1527,6 +1528,32 @@ public function edit_orders_page_actions() {
 			}
 		}
 
+		/**
+		 * Add a "deactivated" message to the plugin list table for WooCommerce Shipping & Tax
+		 *
+		 * @param string $plugin_file
+		 * @param array  $plugin_data
+		 */
+		public function add_custom_message_to_plugin_list( $plugin_file, $plugin_data ) {
+			if ( 'woocommerce-services/woocommerce-services.php' === $plugin_file && false ) {
+				esc_html(
```

Comment:
When doing an sprintf I believe one should escape on the variable side. PHPCS complains about the variable side not being escaped.